packages/app-project/src/components/AuthModal/components/RegisterForm/components/Form/Form.spec.js

```diff
@@ -18,17 +18,23 @@ import {
 } from './Form'
 import en from '../../locales/en'
 
-describe('Component > Form', function () {
+describe('RegisterForm > Component > Form', function () {
   it('should render without crashing', function () {
-    const wrapper = shallow(<Form />)
+    const wrapper = shallow(
+      <Form />,
+      { wrappingComponent: <Grommet />, wrappingComponentProps: { theme: zooTheme } }
+    )
     expect(wrapper).to.be.ok()
   })
 
   describe('fields', function () {
     let rendered
 
     before(function () {
-      const wrapper = shallow(<Form />)
+      const wrapper = mount(
+        <Form />,
+        { wrappingComponent: Grommet, wrappingComponentProps: { theme: zooTheme } }
```

Comment:
Same comment here as to whether we're passing `Grommet` or `<Grommet />` as an option.